src/Schema/Types/Scalars/Upload.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Nuwave\Lighthouse\Schema\Types\Scalars;
+
+use GraphQL\Error\Error;
+use GraphQL\Utils\Utils;
+use Illuminate\Http\UploadedFile;
+use GraphQL\Type\Definition\ScalarType;
+
+class Upload extends ScalarType
+{
+    public $name = 'Upload';
+
+    public $description = ''; // TODO Add description / refer to docs.
+
+    /**
+     * Serialize an internal value, ensuring it is a valid date string.
+     *
+     * @param $value
+     * @throws Error
+     */
+    public function serialize($value)
+    {
+        throw new Error('"Upload" cannot be serialized');
```

Comment:
Do not throw a user facing `Error` here.

Reply:
Should I just throw a `\Exception` for the development related errors, or do you have a special exception for cases like this?

Reply:
We have been using \GraphQL\Error\InvariantViolation from webonyx so far.